template/docs/tutorials/your-first-day.md

```diff
@@ -216,9 +216,26 @@ You can view your change once deployed at `https://<hostname>/humans.txt`
     update this tutorial with anything you think needs fixing up. You can find the
     source for this tutorial at `docs/tutorials/your-first-day.md`
 
+
+## Testing CI Workflows Locally with `act`
+`act` is a tool that allows developers to run their workflows on their local machine without needing to push changes to GitHub. This can significantly speed up the debugging process of CI configurations.
+
+Before you begin, ensure you have `act` installed on your machine. If not, you can install it by following the instructions on the [`act` User Guide][act-user-guide].
+
+To simulate the execution of GitHub Actions workflows on your local machine, use the following command:
+
+```shell
+act push --container-architecture linux/amd64
+```
+
+The `push` parameter triggers the workflow associated with the push event. You can replace push with any other GitHub event that your workflow listens to, such as `pull_request`.
+
+The `--container-architecture` option specifies the architecture of the container in which the workflow will run. linux/amd64 is commonly used, but you may need to adjust this based on your project's requirements or the architecture of your machine.
+
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** This should not be part of someones first day

`act` is a really useful tool for testing github actions. I've used it myself a number of time because no matter what I do I never seem to be able to get a CI config to work first time!

That said, this document is about a developers _first day_ on a project. It is a tutorial aimed at guiding a developer through the steps needed to get the project up and running and making their first contribution. Working on CI pipelines outside of simply triggering them by opening a PR is well outside the scope of a developers first day.

The content is still valuable but I would suggest moving it to it's own tutorial or possible a how-to.

- Tutorials: Step by step guide where the reader is expected to follow the steps verbatim. Assumes little to no familiarity with the system.
- How-To: A more general guide as to help direct the reader's work. Can assume a greater degree of familiarity with the system

For more information of the differences see [The difference between a tutorial and how-to guide](https://diataxis.fr/tutorials-how-to/#). Take a look around that site as well, it details a well thought through approach to documentation and is worth learning about.

Reply:
[TIL] Added a new how-to doc https://github.com/ackama/django-template/commit/bf09b91e1ff0a76efc76667b4756da8cf72aca66